pkg/kfn.go

```diff
@@ -108,6 +108,63 @@ func Build(location string, language languages.Language, imageName string, image
 	return languageManager.BuildImage(systemContext, imageName, imageTag, compiledOutput, additionalFiles, targetDir)
 }
 
+// Build function handles the different build steps:
+// 1. Calculate and create target dir and runtime dir if not existing
+// 2. Resolve the runtime manager
+// 3. Download required runtime files if needed
+// 4. Resolve the compiler manager
+// 5. Check if compile dependencies are available (compiler, libraries, etc)
+// 6. Download on the local filesystem the function if remote
+// 7. Run compilation and output the files to move on the image
+// 8. Resolve image builder
+// 9. Build image and return the image id
+func Test(location string, language languages.Language) error {
```

Comment:
A lot of this function is identical to `kfn.Build()`. Would it be worth extracting a common function that they both can use?